parsl/executors/status_handling.py

```diff
@@ -236,6 +237,13 @@ def _get_block_and_job_ids(self) -> Tuple[List[str], List[Any]]:
     def workers_per_node(self) -> Union[int, float]:
         pass
 
+    def send_monitoring_info(self, status: Dict) -> None:
+        # Send monitoring info for HTEX when monitoring enabled
+        if self.monitoring_radio:
+            msg = self.create_monitoring_info(status)
+            logger.debug("Sending message {} to hub from job status poller".format(msg))
+            self.monitoring_radio.send((MessageType.BLOCK_INFO, msg))
```

Comment:
I recognize it's a code move; do you want to change `job status poller` to something more apropos?